src/checks/kubernetes/deployments/deployment.go

```diff
@@ -102,7 +106,7 @@ func (i inputs) GeneralCheck(kubeClientSet kubernetes.Interface) Results {
 						if inputContainer.Name == container.Name {
 
 							// The number of envars that should exist
-							numberOfEnvars := len(container.Env)
+							numberOfEnvars := len(inputContainer.Env)
```

Comment:
Yeah...oops...was counting the wrong thing.  Should be counting the user input containers =)